frontend/src/pages/Inbox/components/EmailCard.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { css } from 'react-emotion';
+
+import { Tooltip } from 'carbon-components-react';
+
+type Props = {
+  name?: string,
+  photoUrl?: string,
+  role?: string,
+  email?: string,
+  triggerText?: string,
+  direction?: string,
+};
```

Comment:
Awesome that you are typing out the props but if you make them all optional you loose a lot of the benefit of the typing. I would do something like this:
```
type Props = {